README.md

```diff
@@ -7,17 +7,23 @@
 [![license](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
 
-## [Snorkel webpage](http://snorkel.stanford.edu)
+# Quick Links
+* [Snorkel website](snorkel.org)
+* [Snorkel tutorials](snorkel-team/snorkel-tutorials)
+* [Snorkel documentation](https://snorkel.readthedocs.io/en/master/)
+* [Snorkel community forum](https://spectrum.chat/snorkel)
+* [Snorkel mailing list](https://groups.google.com/forum/#!forum/snorkel-ml)
+* [Snorkel twitter](https://twitter.com/SnorkelML)
```

Comment:
maybe add a short section under Community to follow us on Twitter + use #snorkelML ?

Reply:
Twitter capitalized